config/common.go

```diff
@@ -140,3 +156,75 @@ func (cs *commonStore) validate(cfg *model.Config) error {
 
 	return nil
 }
+
+// mergeConfig merges two configs together. The receiver's values are overwritten with the patch's
```

Comment:
See upstream, but we can just use the public `MergeConfig` vs. needing a version on the store.

Reply:
Huh, I didn't see that it had been moved and merged it accidentally. Oops! :)